libraries/MySensors/MyHwATMega328.cpp

```diff
@@ -17,6 +17,8 @@
  * version 2 as published by the Free Software Foundation.
  */
 
+#ifdef __AVR_ATmega328P__
```

Comment:
I agree to that, and my comment is just to give a hint for this error:
http://ci.mysensors.org/job/ArduinoBoardBuilder/a_tool=%2Fopt%2Farduino-1.6.5%2Farduino-headless,arch=avr,board=mega,example=HumiditySensor%2FHumiditySensor.ino,freq=16000000,mcu=atmega2560,pkg=arduino/335/consoleFull#-1860515436253c20e6-3b18-4441-835e-d8f3c24603e1